### ISSUE TITLE
str() takes at most 1 argument (2 given)

### ISSUE THREAD

**Issue by nhatthinh** [incuna/django-wkhtmltopdf#111]
I use Python 2.7.10, Django 1.9.4, django-wkhtmltopdf 3.0.0
Below is the error stack trace:

> Exception Type:	TypeError
> Exception Value:	str() takes at most 1 argument (2 given)
> Exception Location:	wkhtmltopdf/utils.py in http_quote, line 189

```
186.    if isinstance(string, six.text_type):
187.        try:
188.            import unidecode
189.            string = bytes(unidecode.unidecode(string), 'ascii')
190.        except ImportError:
191.            string = string.encode('ascii', 'replace')
192.    # Wrap in double-quotes for ; , and the like
193.    string = string.replace(b'\\', b'\\\\').replace(b'"', b'\\"')
194.    return '"{0!s}"'.format(string.decode())
```

And the value of string is "my_pdf.pdf"
Regarding the error message, we use **bytes** function as a wrong interface.
So, how can I fix this problem?
Thank you so much

**Comment by pauricthelodger** [comment]
Hey @nhatthinh,

I think I might have fixed your problem in #99.
There hasn't been a tagged release since so try installing the master branch and see if that resolves it?

**Comment by nhatthinh** [comment]
Hi Pauricthelodger,
I have resolved this problem.
Thank you so much.
